pkg/controlplane/storageversionhashdata/data.go

```diff
@@ -81,11 +81,11 @@ var GVRToStorageVersionHash = map[string]string{
 	"apps/v1/statefulsets":                                              "H+vl74LkKdo=",
 	"admissionregistration.k8s.io/v1/mutatingwebhookconfigurations":     "Sqi0GUgDaX0=",
 	"admissionregistration.k8s.io/v1/validatingwebhookconfigurations":   "B0wHjQmsGNk=",
-	"admissionregistration.k8s.io/v1/validatingadmissionpolicies":       "P/h9c6yIbaY=",
-	"admissionregistration.k8s.io/v1/validatingadmissionpolicybindings": "XYju31JKYek=",
+	"admissionregistration.k8s.io/v1/validatingadmissionpolicies":       "6OxvlMmQ6is=",
```

Comment:
Why did these storage versions change?

Reply:
To test that this PR correctly resolves mincompatibilityversion from the prerelease information, we removed the hardcoded storage version for VAP (which would have been this release anyway):

https://github.com/kubernetes/kubernetes/blob/91a7708cdcea8cdb1d7db2cc8a27c57741e0cddc/pkg/kubeapiserver/default_storage_factory_builder.go#L74-L75